src/Workspaces/Remote/ServiceHub/Services/CodeAnalysisService_DocumentHighlights.cs

```diff
@@ -16,7 +16,8 @@ internal partial class CodeAnalysisService : IRemoteDocumentHighlights
         {
             var solution = await GetSolutionAsync().ConfigureAwait(false);
             var document = solution.GetDocument(documentId);
-            var documentsToSearch = ImmutableHashSet.CreateRange(documentIdsToSearch.Select(solution.GetDocument));
+            var documentsToSearch = ImmutableHashSet.CreateRange(
```

Comment:
Without this we *definitely* crash as soon as you edit a .cshtml file with js/ts code in it :smile:

See my question in email about how to filter for highlight references.